live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/model/PropertyDelta.java

```diff
@@ -53,9 +54,9 @@ public static <T> PropertyUpdated<T> updated(String key, T propertyValue)
         return new PropertyUpdated<>(key, propertyValue);
     }
 
-    public static ContentPropertyUpdated contentPropertyUpdated(String key, String id, String mimeType)
+    public static ContentPropertyUpdated contentPropertyUpdated(String id, String mimeType)
     {
-        return ContentPropertyUpdated.builder(key).id(id).mimeType(mimeType).build();
+        return ContentPropertyUpdated.builder(CONTENT_PROPERTY).id(id).mimeType(mimeType).build();
```

Comment:
At some point we'll add support for content properties. Currently we only support `cm:content` but custom models can have other properties with a content type.

Reply:
okay, reverted